README.md

```diff
@@ -1,4 +1,29 @@
-# Gateway API Inference Extension 
+# Gateway API Inference Extension
+
+This project offers tools for [AI Inference], enabling developers to build [AI Gateways and Inference Frameworks].
+
+[AI Inference]:https://research.ibm.com/blog/AI-inference-explained
+[AI Gateways and Inference Frameworks]:#concepts-and-definitions
+
+## Concepts and Definitions
+
+AI/ML is evolving rapidly, and [Inference] goes beyond basic networking to include complex traffic routing and optimizations. Below are key terms for developers:
+
+- **GIE**: Short for "Gateway API Inference Extension".
+- **Scheduler**: Manages routing in inference, basing decisions on routing rules (e.g., "model name") and `Metrics and Capabilities` from [Model Serving Platforms].
+- **Metrics and Capabilities**: Data provided by model serving platforms about performance, availability and capabilities to optimize routing. Includes things like [Prefix Cache] status or [LoRA Adapters] availability.
+- **Inference Framework**: The combination of a `Scheduler`, its associated `Metrics and Capabilities`, and an `AI Gateway` is considered an `Inference Framework`.
+- **AI Gateway**: A proxy/load-balancer routing inference traffic. While it is part of an `Inference Framework`, an `AI Gateway` may also operate independently.
```

Comment:
This definition blurs the line between inference gateway and ai gateway as defined by the next section. I think we need to clearly distinguish between the two concepts.

Reply:
Ok, let's pull back for now and we'll think through it some more.

```suggestion
```

Reply:
Tried a different wording, LMKWYT? :thinking: